src/main/java/it/gov/pagopa/payment/repository/TransactionInProgressRepositoryImpl.java

```diff
@@ -14,9 +20,16 @@
 public class TransactionInProgressRepositoryImpl implements TransactionInProgressRepository {
 
   private final MongoTemplate mongoTemplate;
+  private final long trxThrottlingSeconds;
+  private final long trxThrottlingMinutes;
 
-  public TransactionInProgressRepositoryImpl(MongoTemplate mongoTemplate) {
+
+  public TransactionInProgressRepositoryImpl(MongoTemplate mongoTemplate,
+      @Value("${app.qrCode.throttlingSeconds}") long trxThrottlingSeconds,
+      @Value("${app.qrCode.throttlingMinutes}") long trxThrottlingMinutes) {
```

Comment:
The seconds are intended to be a throttling threshold

This minute is intended to represent the duration of a trx to confirm
```suggestion
      @Value("${app.qrCode.trxInProgressLifetimeMinutes}") long trxInProgressLifetimeMinutes) {
```